guides/doc-Creating_a_Standard_Operating_Environment/master.adoc

```diff
@@ -0,0 +1,28 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: soe
+
+= {SOEDocTitle}
+
+// This guide is not ready for stable releases
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+// Step One
+include::common/modules/con_setting-up-your-project-system-management-infrastructure.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-dns-dhcp-and-tftp.adoc[leveloffset=+1]
```

Comment:
```suggestion
:ProductName: {ProjectServer}
include::common/assembly_configuring-dns-dhcp-and-tftp.adoc[leveloffset=+1]

:ProductName: {SmartProxyServer}
include::common/assembly_configuring-dns-dhcp-and-tftp.adoc[leveloffset=+1]
```
Probaby like this?
In any case, you're missing a `:ProductName:` definition for this module and that's why the build fails.